addons/mass_mailing_sms/tests/test_mailing_sms_ui.py

```diff
@@ -0,0 +1,43 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+
+@odoo.tests.tagged('post_install', '-at_install')
+class TestUi(odoo.tests.HttpCase):
+
+    def test_mailing_mailing_activities_split_in_systray(self):
+        """ Activities linked to mailing.mailing records can appear either in the
+            'Email Marketing', either in the 'SMS Marketing' category, depending on
+            the value of the field mailing_type of the record it is linked to. This
+            test ensure that:
+                - activities linked to records with mailing_type set to mail are listed
+                  in the 'Email Marketing' category
+                - activities linked to records with mailing_type set to sms are listed
+                  in the 'SMS Marketing' category
+        """
+
+        # 1. Creation of one sms and one email mailing.mailing records
+        with self.with_user("admin"):
+            sms_rec, email_rec = self.env['mailing.mailing'].create([
+                {
+                    'name': 'SMS record with an activity',
```

Comment:
:socks: When possible, order keys alphabetically (don't cost much to change it :) )